lib/operations/command_v2.js

```diff
@@ -15,13 +15,14 @@ class CommandOperationV2 extends OperationBase {
   constructor(parent, options, operationOptions) {
     super(options);
 
-    const noInherit = this.hasAspect(Aspect.NO_INHERIT_OPTIONS);
-    const inherit = noInherit ? {} : parent;
+    if (this.hasAspect(Aspect.NO_INHERIT_OPTIONS)) {
+      parent = undefined;
+    }
```

Comment:
I think this needs to be moved down below the next line, `    this.ns = parent.s.namespace.withCollection('$cmd');` or something like that, because `parent.s` won't exist anymore now.

Reply:
Reading this after it's been fixed was super confusing. This is out of date, odd it is not labeled.